Frontend/frontend/src/Components/Alert.css

```diff
@@ -0,0 +1,59 @@
+css .display-block {
+  display: block;
+}
+
+.display-none {
+  display: none;
+}
+
+#modal {
```

Comment:
Might be better to also uses classes instead of ids for these. Also the naming convention is inconsistent e.g. `.UserInfomation` vs `userDetailsButton2`